.github/workflows/ci.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       - name: Get latest CMake and ninja
         uses: lukka/get-cmake@latest 
 
+      - name: Install boost library
```

Comment:
A similar change is already in main. Please rebase.

Reply:
Successfully rebased to main and removed the stc bin file